src/guild/cache.rs

```diff
@@ -10,6 +13,7 @@ type QueueMessage = (Message, Arc<RwLock<usize>>);
 #[derive(Default)]
 pub struct GuildCache {
     pub queue_messages: Vec<QueueMessage>,
+    pub current_skip_votes: Vec<UserId>,
```

Comment:
This is better off being a [HashSet](https://doc.rust-lang.org/std/collections/struct.HashSet.html), it'll save you a few checks on the skip command.